README.md

```diff
@@ -136,6 +136,11 @@ By default, we do not show days from the previous month and the next month in th
    initialVisibleMonth: PropTypes.func,
 ```
 
+Optionally you can provide calendar caption block renderer.
+```js
+   renderCalendarCaption: PropTypes.func,
```

Comment:
Glad to see your response!
Maybe its better to this props `renderCalendarInfo`?